packages/ui/src/ui/pages/system/NodeQuad/NodeQuad.tsx

```diff
@@ -17,6 +17,7 @@ interface Props {
         | 'mixed'
         | 'registered'
         | 'unregistered'
+        | 'unvoting'
```

Comment:
Is it correct theme value?


[here](https://github.com/ytsaurus/ytsaurus-ui/pull/134/commits/4d69b22b75fdfc9ae2f632489cbf23da5b561ca0#diff-8112bb582b99aeefe0a4f439f5fcf4aa9d02178a73480ef05239e7fc1a045107R54)
 i see usage as `nonvoting`